src/com.frostycoolslug.goxlr-utility.sdPlugin/app/toggle-mute.js

```diff
@@ -1,5 +1,7 @@
 const muteToggle = new Action('com.frostycoolslug.goxlr-utility.toggle-mute');
 const muteMonitors = {};
+let longPress = null;
+const longPressDelay = 500; // 500ms
```

Comment:
Rather than having `longPressDelay` hard coded, would it be better to pull the configuration setting for hold duration from the status object when the button is pressed? That way the hold duration matches that of the buttons on the GoXLR.

It should be in `status.mixers[payload.settings.serial].settings.mute_hold_duration`, just make sure the serial exists in the mixers first.

Reply:
THAT IS A THING? lol I'll take a look